paasta_tools/metrics/metastatus_lib.py

```diff
@@ -78,8 +79,20 @@ def get_mesos_disk_status(metrics):
     return total, used, available
 
 
+def get_mesos_gpu_status(metrics):
+    """Takes in the mesos metrics and analyzes them, returning gpus status.
+
+    :param metrics: mesos metrics dictionary
+    :returns: Tuple of the output array and is_ok bool
+    """
+    total = metrics['master/gpus_total']
```

Comment:
There are available as far as know. I checked mesosstage, which has never had GPUs and it has them: http://mesos.paasta-mesosstage.yelp:5050/metrics/snapshot